addons/product/views/product_views.xml

```diff
@@ -214,7 +214,7 @@
             <field name="model">product.product</field>
             <field name="mode">primary</field>
             <field name="arch" type="xml">
-                <form string="Variant Information">
+                <form string="Variant Information" duplicate="0">
```

Comment:
According to the doc (and the few times it is used) it should be set to `false` (probably does the same, but let's be consistent).